### DISCUSSION TITLE
SQL Editor - How to Change End-Of-Line (EOL) character?

### DISCUSSION THREAD

**Discussion by treatmesubj** [dbeaver/dbeaver#22395] [Q&A] [answered]
![image](https://github.com/dbeaver/dbeaver/assets/39680353/a72b1d7a-e15f-44d0-a1ea-a177cefced66)
How do I change the end-of-line character in the SQL editor from this paragraph looking symbol to a different character?

**Comment by ShadelessFox** [comment]
It can be changed here:

![image](https://github.com/dbeaver/dbeaver/assets/35821147/1385ca38-5e9e-497d-a65e-2d08ae8142f0)

**Reply by treatmesubj** [reply]
@ShadelessFox thanks, can they be turned off/on?
What are the symbols called? They look like paragraph marks.
It would be cool if we could change the line delimiter to a different character such as `$` like I use in vim.

**Reply by ShadelessFox** [reply, marked as answer]
They can be turned on/off, but they can't be changed:

![image](https://github.com/dbeaver/dbeaver/assets/35821147/0c2c918d-88ea-4bdf-896d-8ed7a11e2f10)
